pkg/cmd/edge_applications/build/build_test.go

```diff
@@ -58,6 +57,6 @@ func TestBuild(t *testing.T) {
 		}
 
 		err := command.run()
-		require.ErrorIs(t, err, utils.ErrorUnsupportedType)
+		require.ErrorContains(t, err, "Error executing Vulcan")
```

Comment:
```suggestion
		require.ErrorContains(t, err, msg.ErrorVulcanExecute)
```

Reply:
Doesn't work, because the message has a placeholder '%s' that will not be contained in the message in stdout/stderr